pyfr/solvers/baseadvec/inters.py

```diff
@@ -94,3 +98,33 @@ def _eval_opts(self, opts, default=None):
             return [npeval(cfg.get(sect, k, default), cc) for k in opts]
         else:
             return [npeval(cfg.get(sect, k), cc) for k in opts]
+
+    def _exp_opts(self, opts, lhs, default=None):
```

Comment:
May be useful.
Say, if we decide u=0 v=0 w=0 by default for the subsonic inflow condition.
Not yet imeplemented, seeking view of others.

Or u=0 for no slip wall by default, but u,v,w
can be set to emulate a moving wall? Well this is usually a constant, but!

On Wed, May 13, 2015 at 12:14 PM, Freddie Witherden <
notifications@github.com> wrote:

> In pyfr/solvers/baseadvec/inters.py
> <https://github.com/vincentlab/PyFR/pull/52#discussion_r30221140>:
>
> > @@ -94,3 +98,33 @@ def _eval_opts(self, opts, default=None):
> >              return [npeval(cfg.get(sect, k, default), cc) for k in opts]
> >          else:
> >              return [npeval(cfg.get(sect, k), cc) for k in opts]
> > +
> > +    def _exp_opts(self, opts, lhs, default=None):
>
> Ditch the default parameter if it is not used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vincentlab/PyFR/pull/52/files#r30221140>.
>



-- 
================================
Dr. Iyer Arvind Sundaram

The reasonable man adapts himself to the world; the unreasonable one
persists in trying to adapt the world to himself.

Therefore all progress depends on the unreasonable man

- George Bernard Shaw.